lib/elixir/lib/code.ex

```diff
@@ -50,9 +50,11 @@ defmodule Code do
 
   ## Examples
 
-      Code.append_path(".") #=> true
+      iex> Code.append_path(".")
```

Comment:
Many of `Code` examples don't use `iex>` intentionally, because of undesirable side-effects.